module/x/gravity/keeper/msg_server_test.go

```diff
@@ -69,34 +69,6 @@ func TestConfirmHandlerCommonWithLowercaseAddress(t *testing.T) {
 	assert.Nil(t, err)
 }
 
-func TestConfirmHandlerCommonWithUppercaseAddress(t *testing.T) {
```

Comment:
This should not be deleted, if it's causing trouble with dupl the common elements should be factored out into their own function. In this case this test is very similar to the one below with just a different string, so the setup can be factored out into it's own function.